### PR TITLE
Deprecate old gitlab

### PR DESCRIPTION
**Pull request by maimorag**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [X] Ready

## Related Issues
fixes: link to the issue

## Description
deprecate GitLab integration. 

## Does it break backward compatibility?
   - [X] No

### CODE REVIEW

**Review by Shellyber — COMMENTED** [review]
Looks good.
Let's change the pack version to 1.1.1 (and the release notes file name to match)

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 category: Utilities
 commonfields:
   id: GitLab
-  version: -1
+  version: 1.1.1
```
Comment (Shellyber) [inline Packs/DevSecOps/Integrations/GitLab/GitLab.yml]: revert this

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml [hunk 2]
```diff
@@ -24,22 +24,20 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: An integration with GitLab.
-display: GitLab
+description: Deprecated. Use GitLab Pack instead.
```
Comment (Shellyber) [inline Packs/DevSecOps/Integrations/GitLab/GitLab.yml:27]:
```suggestion
description: Deprecated. Use GitLab v2 in GitLab Pack instead.
```

File: Packs/GitLab/Integrations/GitLab/1_d0445cb2-f325-4894-83e1-93ac42293504 [hunk 3]
```diff
@@ -0,0 +1 @@
+I am a file.
```
Comment (Shellyber) [inline Packs/GitLab/Integrations/GitLab/1_d0445cb2-f325-4894-83e1-93ac42293504:1]: Remove this

**Review by Shellyber — CHANGES_REQUESTED** [review]

**Comment by Shellyber** [comment]
Also don't forget to change the pack version (in the pack_metadata file) and the release notes file name accordingly

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml (hunk 1 above)
Comment (maimorag) [reply Packs/DevSecOps/Integrations/GitLab/GitLab.yml]: done

File: Packs/GitLab/Integrations/GitLab/1_d0445cb2-f325-4894-83e1-93ac42293504 (hunk 3 above)
Comment (maimorag) [reply Packs/GitLab/Integrations/GitLab/1_d0445cb2-f325-4894-83e1-93ac42293504]: ok

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml (hunk 1 above)
Comment (maimorag) [reply Packs/DevSecOps/Integrations/GitLab/GitLab.yml]: ok ok

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml (hunk 2 above)
Comment (maimorag) [reply Packs/DevSecOps/Integrations/GitLab/GitLab.yml]: done

File: Packs/DevSecOps/ReleaseNotes/1_1_1.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitLab
+- Deprecated. Use GitLab v2 Pack instead.
```
Comment (Shellyber) [inline Packs/DevSecOps/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Deprecated. Use GitLab v2 in GitLab Pack instead.
```

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml [hunk 5]
```diff
@@ -24,22 +24,20 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: An integration with GitLab.
-display: GitLab
+description: Deprecated. Use GitLab v2 Pack instead.
```
Comment (Shellyber) [inline Packs/DevSecOps/Integrations/GitLab/GitLab.yml:27]:
```suggestion
description: Deprecated. Use GitLab v2 in GitLab Pack instead.
```

**Review by Shellyber — COMMENTED** [review]
Good.
This PR will be merged only after the GitLab v2 is merged,

File: Packs/DevSecOps/Integrations/GitLab/GitLab.yml (hunk 5 above)
Comment (maimorag) [reply Packs/DevSecOps/Integrations/GitLab/GitLab.yml:27]: done

File: Packs/DevSecOps/ReleaseNotes/1_1_1.md (hunk 4 above)
Comment (maimorag) [reply Packs/DevSecOps/ReleaseNotes/1_1_1.md:4]: done

**Review by Shellyber — APPROVED** [review]

**Pull request merged**